include/modem/modem_info.h

```diff
@@ -123,6 +123,28 @@ extern "C" {
 			       (((float)(rsrq) - 40) * 0.5f) : \
 			       (((float)(rsrq) - 41) * 0.5f)))
 
+/** @brief Converts SNR index value returned by the modem to dB.
+ *
+ * The index value of SNR can be converted to dB with the following formula:
+ * * index – 25
```

Comment:
This should probably be 24, as that is the value used in `SNR_OFFSET_VAL` and in the modem AT command specification.
```suggestion
 * * index – 24
```